src/metabase/models/setting.clj

```diff
@@ -370,11 +370,14 @@
 
 (defn- user-facing-info [setting]
   (let [k (:name setting)
-        v (get k)]
+        v (get k)
+        set-by-env (not (nil? (env-var-value setting)))]
     {:key         k
      :value       (when (and (not= v (env-var-value setting))
                              (not= v (:default setting)))
                     v)
+     :envsetting  set-by-env
```

Comment:
it would be more idiomatic here too to call this something like `is_env_setting` to make it clear we're talking about a boolean here